### PR TITLE
tests: fix preseed test to used core20 snap on latest systems

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Images are using core20 on systems 20.10 and 21.04

### CODE REVIEW

File: tests/main/preseed/task.yaml [hunk 1]
```diff
@@ -42,6 +42,12 @@ restore: |
   umount_ubuntu_image "$IMAGE_MOUNTPOINT" || true
 
 execute: |
+  if os.query is-focal; then
```
Comment (mvo5) [inline tests/main/preseed/task.yaml:45]: Maybe a small comment here ? it's a bit counter-intuitive that focal (20.04) is using core18, maybe some context (if possible) why 20.10+ is using core20

**Review by mvo5 — APPROVED** [review]
Thank you! Just one question/suggestion about a comment.

File: tests/main/preseed/task.yaml (hunk 1 above)
Comment (pedronis) [reply tests/main/preseed/task.yaml:45]: I think it's lxd changing base, these images have lxd in them

Comment (pedronis) [reply tests/main/preseed/task.yaml:45]: maybe we can extract the info from the model inside them at some point, not sure

**Review by pedronis — APPROVED** [review]
thank you

**Review by stolowski — APPROVED** [review]
Thank you!

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**